advanced_samples/Bulk_Upload/DotNet/BulkUploader/Program.cs

```diff
@@ -0,0 +1,219 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OSIsoft.Data;
+using OSIsoft.DataViews.Contracts;
+using OSIsoft.Identity;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using OSIsoft.DataViews;
+using System;
+
+namespace Bulk_Uploader
+{
+    public class Program
+    {
+        public static string dataviewPath;
+        public static string sdsTypePath;
+        public static string sdsStreamPath;
+        public static string sdsStreamMetaPath;
+        public static string sdsStreamTagPath;
+        public static string sdsStreamDataPath;
+        public static string sdsDataOnlyPath;
+
+        public static Exception toThrow = null;
+        public static bool success = true;
```

Comment:
I would delete the `success` variable and instead use `toThrow == null`, they should resolve to the same boolean value.